docs/api-reference/tensorflow-usage.md

```diff
@@ -22,8 +22,20 @@ As of now TensorFlow does not support running on GPUs for MacOS, so we cannot su
 You must have at least one CUDA compatible GPU, for a list of compatible GPUs see
 [Nvidia's Guide](https://developer.nvidia.com/cuda-gpus).
 
-Install [CUDA v10.0](https://developer.nvidia.com/cuda-10.0-download-archive) and [CUDNN v7.6.4](https://developer.nvidia.com/rdp/cudnn-download)
-following [Nvidia's Install guide](https://docs.nvidia.com/cuda/cuda-quick-start-guide/index.html).
+Install [CUDA v10.0](https://developer.nvidia.com/cuda-10.0-download-archive) and [CUDNN v7.6.4](https://developer.nvidia.com/rdp/cudnn-download). 
+
+Make sure you install CUDA v10.0, not any other newer version.
+After downloading CUDNN v7.6.4 .zip file and unpacking it, you need to do the following steps:
+
+`copy <CUDNN_zip_files_path>\cuda\bin\cudnn64_7.dll to C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\bin`
```

Comment:
You are assuming it will "C:" drive, can we please make this generic?

Reply:
Sure, I can change that, but *anyone* using Windows would understand that path based on C:\, anyways. Not a big deal.
That's the way NVIDIA/CUDA/CUDNN defined that step in their documentation, by the way...